### ISSUE TITLE
Movies not appearing in list but do exist

### ISSUE THREAD

**Issue by lobianco** [Radarr/Radarr#1291]
**Description:**

First off, Radarr is awesome and I'm super appreciative of all the work put into it!

Today, all of a sudden, my movies stopped appearing in the home page list. But they do exist (as you can tell by the footer stats) and I can still navigate directly to individual movie pages. The only change I made surrounding the event was an update to the movie folder syntax in Settings. 

Screenshot of the problem:

<img width="1223" alt="screen shot 2017-03-30 at 5 56 50 pm" src="https://cloud.githubusercontent.com/assets/1753971/24527957/719de680-1572-11e7-888d-9b03531b8991.png">

**Radarr Version:**

0.2.0.535

**Logs:**

Nothing in the logs that will help (just old parsing warnings).

**Comment by theCheek** [comment]
@lobianco could you post a screenshot of the movie folder syntax that you made?

**Comment by galli-leo** [comment]
@lobianco Did you filter by something where the movies where not included (e.g. filtered by monitored?)

**Comment by cody00** [comment]
Same exact issue.  None of the movies are showing although are there and showing in stats

**Comment by scooterbaga** [comment]
Same problem.

I've noticed that using the Update Library function will cause everything to show back up. However, it's temporary.

**Comment by lobianco** [comment]
@theCheek I tried changing the movie folder format to `{Original Title}` just to see if it would work even though it's not one of the listed options. After hitting Save the example text was cleared so I decided to undo the change. 

@galli-leo no I was filtered on `All`. 

@scooterbaga interesting, where is the Update Library button?

**Comment by galli-leo** [comment]
Did you try to clear your cache?

**Comment by lobianco** [comment]
I did after I first noticed the problem and it didn't help anything, but I logged on today and all my movies are back in the list so.... I don't know what to think 😆

**Comment by SippieCup** [comment]
I was able to reproduce with two different hosts and saving to a NFS share.

host 1: Has functioning Library
host 2: new install
shared folder: /mnt/pool/media

1. Backup the configuration of radarr on host 1.
3. delete movie folders in /mnt/pool/media
4. restore configuration on to host 2.

**Comment by j-vizcaino** [comment]
Stumbled on this problem as well today. Clearing cookies associated with the hostname did the trick.
It appears cookies set by other services running on the same host (but a different port) confuses the backend and the REST request for movies simply returns and empty list.

**Comment by galli-leo** [comment]
@j-vizcaino Can you provide an example of such a request as well as the cookies in question? Radarr should store all cookies with a Radarr prefix

**Comment by j-vizcaino** [comment]
I cleared the cookie to make this work :( The thing I remember from the network capture in Chrome was that the cookie was around 300 chars long and totally unrelated to Radarr (it had some prefix with some torrent helper).
Will try to reproduce the problem but can't promise anything. In the meantime I hope that helps a little.

**Comment by mattpurnell** [comment]
Confirm that the issue is occurring on a new Radarr docker DL:

Version
0.2.0.1120
Mono Version
5.12.0.226 (tarball Thu May 3 09:48:32 UTC 2018)

**Comment by deurk** [comment]
Just noticed the issue on same version than above. Movies are here, any search will return the correct results, there are just not displayed on homepage anymore.

Using Docker image, 2.0.0.5228, NFS mounted shares.

**Comment by f8k8** [comment]
I think this happens when you've set a filter on the movies page (such as `Missing Only`) which then later ends up with no results. E.g.:
- Set it to missing only
- Close the browser and wait for your downloads to complete so that you no longer have any missing ones
- Open Radarr again and go to the movies page

Radarr will remember the filter for Missing Only and return 0 results (as they're all downloaded now). And because there's 0 results, it'll show you the "You must be new around here" message without being able to change the filters.

A workaround is to add / edit a movie so that it matches whatever filter you last specified. E.g. if you were on Missing Only, just add a random movie to your list. Then go back to the movies page and the list and filters will be shown and you can reset the filter and remove the movie you just added.

**Comment by SuperFlea2828** [comment]
RESOLVED - Same issue. Resolved by deleting cookie using Chrome developer tools, then closing and reopening tab using bookmark. Prior to issue I was updating library. I tried running Refresh Movie task and it did make the first movie appear briefly in the listing but it would quickly disappear. 

Chrome: 
  Version 75.0.3770.100 (Official Build) (64-bit)
Radarr:
  Version 0.2.0.1217
  Mono Version 5.16.0.179 (tarball Thu Oct 4 10:29:10 UTC 2018)
Docker:
  Image: linuxserver/radarr
